src/org/purefn/egon/interop.clj

```diff
@@ -101,6 +102,11 @@
          (.setContentType "application/edn")
          (.setContentLength (alength bytes)))])))
 
+(defn get-request [bucket key version-id]
+  (if version-id
+    (GetObjectRequest. bucket key version-id)
```

Comment:
Calling the constructor with version-id as nil works fine, but is not documented so adding condition here.